### DISCUSSION TITLE
Unable to merge branch with main on Github because of unresolved conflicts

### DISCUSSION THREAD

**Discussion by Manuella-Sam** [neuefische/questions#688] [Web] [answered]
I'm getting the message "This branch has conflicts that must be resolved" on Github while trying to merge the branch I previously worked on.I don't understand where the problem lies.
Here is the [PR](https://github.com/Manuella-Sam/dont-forget-to-change-the-project-name/pull/14).

Thank you in advance.

**Comment by Kheirah** [comment, marked as answer]
The conflict is due to changes [you made directly](https://github.com/Manuella-Sam/dont-forget-to-change-the-project-name/compare/feature/job-seeking-page...main) on the `main` branch.

You will need to resolve it before merging. Decide which parts to keep and which to discard. If you need a reminder how to solve merge conflicts, check the session on Advanced Git. You can solve the conflict directly on GitHub.
